Client/Frontend/Fakespot/FakespotViewController.swift

```diff
@@ -193,6 +199,6 @@ class FakespotViewController: UIViewController, Themeable {
 
     @objc
     private func closeTapped() {
-        dismissVC()
+        delegate?.fakespotControllerCancelButtonTapped()
```

Comment:
Hey, sorry it slipped by me. Indeed there's a way to dismiss it by tapping the button and I'm not sure if there are extra requirements at this point to be dismissed in other ways. Because we agreed to use the detents API there is additional swipe and tapping outside so I'm updating the code to take that into account.